src/selfie_control/src/ChangeLane.cpp

```diff
@@ -0,0 +1,53 @@
+#include "ChangeLane.h"
+
+ChangeLane::ChangeLane(float lane_width_param, float error_margin_param):
+    lane_width(lane_width_param),
+    error_margin(error_margin_param),
+    target_position(lane_width_param/2)
+{
+}
+
+ChangeLane::~ChangeLane()
+{
+}
+
+//main fuction for computing value for target position
+//input lane_width, error_margin, position_offset
+//output taget_positions, left_turn_indicator, right_turn_indicator
+//left +, right -
+bool ChangeLane::process_target_position(void){
+    if (be_on_left_lane == true){
```

Comment:
Consider instead: `if (be_on_left_lane)`. This way it is a bit closer to natural language.